plugins/smart_proxy_reports/control

```diff
@@ -0,0 +1,23 @@
+Source: ruby-smart-proxy-reports
+Section: ruby
+Priority: optional
+Maintainer: Lukas Zapletal <lzap+pub@redhat.com>
+Build-Depends: debhelper (>= 11~),
+               ruby-rack (>= 1.1.0),
+               gem2deb
+Standards-Version: 4.3.0
+Homepage: http://github.com/theforeman/smart_proxy_reports
+Testsuite: autopkgtest-pkg-ruby
+XS-Ruby-Versions: all
+Rules-Requires-Root: no
+
+Package: ruby-smart-proxy-reports
+Architecture: all
+XB-Ruby-Versions: ${ruby:Versions}
+Depends: ruby | ruby-interpreter,
+         foreman-proxy (>= 2.5.0),
+         ruby-rack (>= 1.1.0),
```

Comment:
this is clearly wrong:

https://github.com/theforeman/smart_proxy_reports/blob/ce29f1b57b0090945e8c999debf9a58cd7390331/smart_proxy_reports.gemspec#L18-L19